packages/webcomponents/src/components/table/utils.tsx

```diff
@@ -18,8 +18,8 @@ export const EmptyState = (columnSpan: number) => (
 
 export const ErrorState = (columnSpan: number, errorMessage: string) => (
   <tr>
-    <td class="error-state" part="error-state" colSpan={columnSpan} style={{ textAlign: 'center' }}>
+    <td class="error-state" part="error-state" colSpan={columnSpan} data-test-id="error-state" style={{ textAlign: 'center' }}>
```

Comment:
Very small thing - since this error state is specifically for the table, could we rename this to `data-test-id='table-error-state'` ?